kolibri/core/discovery/well_known.py

```diff
@@ -1,9 +1,14 @@
 from le_utils.uuidv5 import generate_ecosystem_namespaced_uuid
 
+from kolibri.core.serializers import HexOnlyUUIDField
 from kolibri.utils import conf
 
 # AKA Kolibri Studio
 CENTRAL_CONTENT_BASE_URL = conf.OPTIONS["Urls"]["CENTRAL_CONTENT_BASE_URL"]
 CENTRAL_CONTENT_BASE_INSTANCE_ID = generate_ecosystem_namespaced_uuid(
     CENTRAL_CONTENT_BASE_URL
 ).hex
+
+# AKA Kolibri Data Portal
+DATA_PORTAL_SYNCING_BASE_URL = conf.OPTIONS["Urls"]["DATA_PORTAL_SYNCING_BASE_URL"]
+DATA_PORTAL_BASE_INSTANCE_ID = HexOnlyUUIDField()
```

Comment:
I'm confused by this. When I thought of 'hard coding the ID' I thought of the actual ID. Would you explain how this works?

Reply:
I'm glad you called this out, I completely used that incorrectly! I could actually use clarification on “the actual ID.” Is this the known ID of KDP? I am not sure what that ID is and assumed that a hardcoded ID needed to be created, I’m realizing now that `HexOnlyUUIDField()` is very obviously a class and not a function!

Reply:
By actual ID, I mean the `instance_id` returned by its API `/api/public/info/`. That shouldn't change, unless you configure a development instance of Kolibri to use an alternate KDP instance. The approach for Studio works well because the ID is based off the URL, so with a custom URL the ID should end up matching it. It would be nice if we could do that for KDP, but there are repercussions of that in regards to syncing which I'm not sure we want to tackle.

We really don't have to be strict about the ID matching at this time, or at least until we have a good reason to.